src/test/java/org/broadinstitute/hellbender/utils/IntervalUtilsUnitTest.java

```diff
@@ -608,6 +608,13 @@ public void testUseAllHG38ContigsInIntervalQuery() {
         );
     }
 
+    @Test(expectedExceptions = org.broadinstitute.hellbender.exceptions.UserException.class)
+    public void testReasonableErrorWithInvalidBedFile() throws IOException {
+        File outOfBoundsBedFile = new File(INTERVAL_TEST_DATA + "bad_interval_span.bed"); // this bed file has a bogus interval the extends past the end of the
+        // this should throw a user exception because of the invalid interval
+        IntervalUtils.featureFileToIntervals(hg19GenomeLocParser, outOfBoundsBedFile.getAbsolutePath());
+    }
```

Comment:
Add similar tests with an out-of-bounds interval in Picard and GATK format interval files

Reply:
added